demos/src/Extensions/FontFamily/React/index.jsx

```diff
@@ -79,13 +79,6 @@ export default () => {
           >
             CSS variable
           </button>
-          <button
```

Comment:
I have removed this `button` since I don't see any utility into it and does not apply for any of the cases that we're showcasing

Reply:
👍 yep